.github/workflows/test-linux.yml

```diff
@@ -124,16 +116,10 @@ jobs:
     - name: Bootstrap Action Workspace
       uses: ./.github/workflows/bootstrap
     
-    - name: Build agent-zip
-      run: ./build.sh agent-zip
-
     - name: 'Tests: StartupHooks'
-      uses: ./.github/workflows/test
-      with:
-        name: 'startuphooks'
-        project: 'test/startuphook/Elastic.Apm.StartupHook.Tests/Elastic.Apm.StartupHook.Tests.csproj'
-        
-
+      shell: bash
+      run: ./build.sh test --test-suite StartupHooks
```

Comment:
`StartupHooks` should be lower-cased instead? I see that pattern in the other places when using `--test-suite`